iac-extensions/docker/src/main/java/org/sonar/iac/docker/checks/utils/CheckUtils.java

```diff
@@ -61,4 +62,12 @@ public static BiConsumer<CheckContext, RunInstruction> ignoringHeredoc(BiConsume
       }
     };
   }
+
+  public static <T extends CommandInstruction> BiConsumer<CheckContext, T> ignoringExecForm(BiConsumer<CheckContext, T> visitor) {
```

Comment:
Please add Javadoc, when it makes sense to ignore ExecForm, to not forget next time